### PR TITLE
fix err links:https://github.com/kubernetes-sigs/cluster-api-provider…

### PR DESCRIPTION
**Pull request by ttonline6**
…-aws/blob/release-0.2/docs/getting-started.md

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/roleassumption.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Using clusterctl command we can create a new cluster in AWS which in turn will a
 management cluster to create the target cluster(in a different AWS account. This can be achieved by using the phases in
 clusterctl to perform all the steps except the pivoting. This will provide us with a bare-bones functioning cluster that
 we can use as a management cluster.
-To begin with follow the steps in this [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/docs/getting-started.md) to setup the environment
+To begin with follow the steps in this [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/release-0.2/docs/getting-started.md) to setup the environment
```
Comment (randomvariable) [inline docs/roleassumption.md]:
```suggestion
To begin with follow the steps in this [getting started guide](https://cluster-api.sigs.k8s.io/user/quick-start.html) to setup the environment
```
is the real getting started guide

**Comment by detiber** [comment]
@ttonline6 can you update with the suggestions from @randomvariable ?

**Comment by vincepri** [comment]
@ttonline6 seems your original commit doesn't have a user attached to it

**Comment by detiber** [comment]
@ttonline6 are you able to update this as requested?

**Comment by randomvariable** [comment]
@ttonline6 feel free to re-open if you get CLA auth sorted out
/close

**Pull request closed without merging**